docs/remote_inference_blueprints/bedrock_connector_ai21labs_jurassic_blueprint.md

```diff
@@ -0,0 +1,171 @@
+# Bedrock connector blueprint example for AI21 Labs Jurassic-2 Mid model
+
+## 1. Add connector endpoint to trusted URLs:
+
+Note: no need to do this after 2.11.0
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
+
+## 2. Create connector for Amazon Bedrock:
+
+If you are using self-managed Opensearch, you should supply AWS credentials:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+    "name": "Amazon Bedrock",
+    "description": "Test connector for Amazon Bedrock",
+    "version": 1,
+    "protocol": "aws_sigv4",
+    "credential": {
+        "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+        "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>"
+    },
+    "parameters": {
+        "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+        "service_name": "bedrock",
+        "model_name": "ai21.j2-mid-v1"
+    },
+    "actions": [
+        {
+            "action_type": "predict",
+            "method": "POST",
+            "headers": {
+                "content-type": "application/json"
+            },
+            "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model_name}/invoke",
+           "request_body": "{\r\n\"prompt\":\"${parameters.inputs}\",\r\n\"maxTokens\":200,\r\n\"temperature\":0.7,\r\n\"topP\":1,\r\n\"stopSequences\":[\r\n\r\n],\r\n\"countPenalty\":{\r\n\"scale\":0\r\n},\r\n\"presencePenalty\":{\r\n\"scale\":0\r\n},\r\n\"frequencyPenalty\":{\r\n\"scale\":0\r\n}\r\n}",
```

Comment:
Can we remove `\r\n`?

Reply:
Sure thing. I just pushed another commit with less escaping metadata. I tested before commit and it works just fine.